### PR TITLE
Leaderboard

### PR DESCRIPTION
**Pull request by spraveenofficial**
Added LeaderBoard
Added Confetti (It will be visible for who's leading the table)

### CODE REVIEW

File: src/Redux/Actions/leaderboard.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    });
+    const token = localStorage.getItem("token");
+    const { data } = await axios({
+      method: "GET",
+      url: `${baseUrl}/leaderboard`,
+      headers: {
+        token: `Bearer ${token}`,
+      },
+    });
+    // const decryptedData = await decrypt(data.data);
```
Comment (shivanigangadharan) [inline src/Redux/Actions/leaderboard.js:21]: Can remove comment if not wanted

**Review by shivanigangadharan — APPROVED** [review]
I like how you've added suitable comments for functions. Code looks good 👍🏼

**Review by ajay-mandaviya — APPROVED** [review]
Everything looks to good , with well structure
you can also make different folder for your every redux state 
with actionTypes , action and  reducer for every state

**Comment by spraveenofficial** [comment]
> Everything looks to good , with well structure you can also make different folder for your every redux state with actionTypes , action and reducer for every state

Yes ajay, i always use that way only. But as this this mini app. So have written all the constants in one file only.

**Pull request merged**